src/components/Minibrowser/components/MinibrowserListContainer.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="sb-minibrowser__list-container">
+    <SbMinibrowserBreadcrumbs
+      v-if="hasBreadcrumbs"
+      :items="navigationItems"
+    />
+
+    <template v-if="hasGroupItems">
+      <SbMinibrowserList
+        v-for="(groupItem, index) in groupItems"
+        :key="index"
+        v-bind="groupItem"
+      />
+    </template>
+
+    <template v-if="hasOtherItems">
```

Comment:
Why not use direct v-if in the component?

Reply:
Yeah, you're right. Thanks!